docs/learning-powershell/powershell-beginners-guide.md

```diff
@@ -305,7 +305,6 @@ For more details, see [Create and Run PowerShell Script Guide][create-run-script
 
 - Video: [Get Started with PowerShell][remoting] from Channel9
 - [eBooks from PowerShell.org](https://leanpub.com/u/devopscollective)
-- [eBooks from PowerShell.com][ebooks-powershell.com]
```

Comment:
I don't see the point of duplicating the link in two markdown docs, so just keeping it in the readme.md